src/Tickets/Commerce/Gateways/PayPal/Ajax_Request_Handler.php

```diff
@@ -170,42 +156,42 @@ public function onGetPartnerUrlAjaxRequestHandler() {
 	 *
 	 * @since 5.1.6
 	 */
-	public function removePayPalAccount() {
-		$this->validateAdminRequest();
+	public function remove_paypal_account() {
+		$this->validate_admin_request();
 
 		// Remove the webhook from PayPal if there is one
-		if ( $webhookConfig = $this->webhooksRepository->getWebhookConfig() ) {
-			$this->webhooksRepository->deleteWebhook( $this->merchantDetails->accessToken, $webhookConfig->id );
-			$this->webhooksRepository->deleteWebhookConfig();
+		if ( $webhook_config = $this->webhooks_repository->get_webhook_config() ) {
+			$this->webhooks_repository->delete_webhook( $this->merchant->get_access_token(), $webhook_config->id );
+			$this->webhooks_repository->delete_webhook_config();
 		}
 
-		$this->merchantRepository->delete();
-		$this->merchantRepository->deleteAccountErrors();
-		$this->merchantRepository->deleteClientToken();
-		$this->refreshToken->deleteRefreshTokenCronJob();
+		$this->merchant->delete_data();
+		$this->merchant->delete_account_errors();
+		$this->merchant->delete_access_token_data();
+		$this->refresh_token->delete_refresh_token_cron_job();
 
 		wp_send_json_success( __( 'PayPal account disconnected', 'event-tickets' ) );
 	}
 
 	/**
 	 * Create order.
 	 *
-	 * @since 5.1.6
-	 * @todo : handle payment create error on frontend.
+	 * @todo  : handle payment create error on frontend.
 	 *
+	 * @since 5.1.6
 	 */
-	public function createOrder() {
+	public function create_order() {
 		// @todo Set up the order with our own custom code.
 
-		$this->validateFrontendRequest();
+		$this->validate_frontend_request();
 
 		$postData = give_clean( $_POST );
 		$formId   = absint( tribe_get_request_var( 'give-form-id' ) );
```

Comment:
Does this form ID need a change?

Reply:
We will, I've added the proper todos to make sure that happens.